### DISCUSSION TITLE
请问以下步骤需要改进些什么嘛

### DISCUSSION THREAD

**Discussion by AsuKna** [mikumifa/biliTickerBuy#296] [提问专区]
间隔300，输入rrcour的密钥后提前四分钟开始抢。大概过了十秒弹出可以验证的选项，验证完就点击按钮，这个时候后台应该就启动了
如果用第四个方法本地的话是怎么操作，下载了软件但是后缀pyi不知道怎么打开（python好像只能打开py后缀的），是怎么开启的呢？
![1e668f9890e4a7f7a26b90abeee47f8](https://github.com/mikumifa/biliTickerBuy/assets/174230504/1e4b0c07-78f6-4072-b7f3-918c1d8ee6fa)

正常来说抢到了是自动提交订单但不付款，再用手机付款吗？
![image](https://github.com/mikumifa/biliTickerBuy/assets/174230504/87943083-7a01-4d3c-8d8d-0cd208410a2a)
![1e668f9890e4a7f7a26b90abeee47f8](https://github.com/mikumifa/biliTickerBuy/assets/174230504/406d51a6-dfbb-47ad-8df2-97f4d6517b5b)
![e912baa5aac3845dc7801353d7d94ef](https://github.com/mikumifa/biliTickerBuy/assets/174230504/7cbfeb60-7175-4ce0-aa97-65b04f466d9d)

**Comment by mikumifa** [comment]
本地验证码是最优解，rrocr到抢票时候很多人用，速度不会特别快

本地验证码实在没办法使用的话，记得rrocr用的时候可以同时手过验证码，所以`time>=min(rrocr_time, 手动时间)`

**Comment by mikumifa** [comment]
如果没有显示本地验证码（第四个选项），那就是系统不支持
